app/views/posts/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<a href="<%= users_path() %>">
+  <button class="custom-back-btn">⬅</button>
+</a>
+<ul>
+  <a href="<%= user_path(@user) %>">
+    <div class="custom-user-box">
+      <%= render partial: 'user_helper/user_photo', locals: { user: @user } %>
+      <%= render partial: 'user_helper/user_details', locals: { user: @user } %>
+    </div>
+  </a>
+  <% @posts.each do |post|%>
+    <a href="<%= user_post_path(@user, post) %>">
+      <div class="custom-user-post">
+        <div class="custom-post">
```

Comment:
- [ ] To ensure the view files run smoothly, please instantiate the @post and @user variables in the Post controller. Without these, the view may encounter errors. Here’s how you can define them:

```rb
# In the Post controller
def index
  @user = User.find(params[:user_id])
  @posts = Post.includes(:comments).where(author_id: params[:user_id])
end

```

Reply:
This change has been done well